Moderation.md

```diff
@@ -0,0 +1,23 @@
+# Moderation Overview
+
+The Windows Package Manager community repository is the location for manifests published in the default Windows Package Manager source. Automated pipelines validate and publish these manifests. The quality of the metadata has a direct impact on customer experience. Human review has proven to be an invaluable tool for ensuring we can provide the best experiences.
+
+## Moderators
+
+In addition to Microsoft employees, several community members have been identified through their high-quality submissions, willingness to help others, and adherence to our [code of conduct](CODE_OF_CONDUCT.md). You can see who they are in a pinned [public service announcement](https://github.com/microsoft/winget-pkgs/issues/15674).
+
+When we initially implemented moderation, we observed several objective criteria. These included the number of Pull Requests (PR)s made, the length of time they had been active in the project, and their interaction with others who submitted PRs.
+
+Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our goal from now on is to ensure the community is sufficiently supported by individuals who have the technical knowledge and a proven track record of success. Becoming a moderator is like becoming a [Microsoft Most Valuable Professional](https://mvp.microsoft.com/Pages/what-it-takes-to-be-an-mvp). There isn't a set formula. We're looking at what individual contributors are doing, and how they are doing it. If the need arises to add additional moderators, nominations may come from one of the core team members or an existing moderator. We will publicly disclose the nomination in a discussion. If the nominee agrees to the requirements in this document, they may be awarded the status of moderator.
```

Comment:
```suggestion
Our intent was not to introduce a numbers game for others to achieve and automatically become a moderator. Our goal from now on is to ensure the community is sufficiently supported by individuals who have the technical knowledge and a proven track record of success. Becoming a moderator is like becoming a [Microsoft Most Valuable Professional](https://mvp.microsoft.com/en-us/Pages/what-it-takes-to-be-an-mvp). There isn't a set formula. We're looking at what individual contributors are doing, and how they are doing it. If the need arises to add additional moderators, nominations may come from one of the core team members or an existing moderator. We will publicly disclose the nomination in a discussion. If the nominee agrees to the requirements in this document, they may be awarded the status of moderator.
```
Looks like this page is broken for me without the `en-us`:
![image](https://user-images.githubusercontent.com/11600822/131386677-bf4b8748-39b2-45bf-92f6-58b9e5f1e0dd.png)

I'm in Europe (United Kingdom - en-GB). Not sure if anyone else is running into this.

Reply:
I'm receiving the same error. Seems to only be available in following languages:

![image](https://user-images.githubusercontent.com/52826317/131387076-9e0ac885-8ca1-4d27-982d-b50670d3db98.png)

Reply:
It still errors even if you select one of those languages 🤦‍♂️

Reply:
Note the `ru-RU` in URL
![image](https://user-images.githubusercontent.com/83997633/131389932-45132053-1190-4261-b788-d832f3bab7ef.png)

Reply:
Works with `de-DE` though...
![image](https://user-images.githubusercontent.com/83997633/131390154-e1982def-41e0-4054-a9ae-258d04cf6bff.png)

Reply:
> Works with `de-DE` though...

`de-DE` is Deutsch/German which is listed on the website. 

The languages themselves work, but the URL without the language codes does not redirect/display correctly (if it's even supposed to, I would assume so as some of the URLs does display the correct language that you select without needing the language codes in the URL). Seems to only be an issue with some URLs on their website.

`en-US` is obviously the best option here like @ItzLevvie suggested. Ideally, the website should be updated and follow the same style (or similar) across the board as Microsoft Docs where you get redirected to `en-US` if the language is not listed, or you get redirected to the language you have already selected on that site.